package/com.unity.formats.usd/Runtime/Scripts/IO/Geometry/MeshImporter.cs

```diff
@@ -225,7 +225,7 @@ public class GeometrySubsets {
                              SceneImportOptions options) {
       var smr = ImporterBase.GetOrAddComponent<SkinnedMeshRenderer>(go);
       if (smr.sharedMesh == null) {
-        smr.sharedMesh = new Mesh();
```

Comment:
I wonder if there's a more meaningful name we could give here (just thinking out loud, this is not a request to change it)

Reply:
I agree. Call it something maybe like go.name+"_dyn_mesh" or whatever.

Reply:
Did that before. But that doesn't help if you want to know if the mesh is shared or not.
I'll probably go for something like: go.name + guid.

Reply:
Done